scripts/deployBridge.js

```diff
@@ -131,6 +192,9 @@ async function main() {
   await tx8.wait();
   console.log("Transaction hash: ", tx8.hash);
   console.log("--------------------------------------------------------------");
+  console.log("Sleeping for 30s...")
+  await sleep(30000);
```

Comment:
create a variable for this and put it at the top. 30 seconds is not enough for all networks. some networks take more time or reorg. its difficult to change all the numbers one by one.